deny.toml

```diff
@@ -8,56 +8,49 @@ multiple-versions = "deny" #catch mutliple versions of a crate
-    {name = "digest"},
-    {name = "getrandom"},
-    {name = "heck"},
-    {name = "itoa"},
-    {name = "parking_lot" },
-    {name = "parking_lot_core" },
-    {name = "pin-project"},
-    {name = "pin-project-internal"},
-    {name = "pin-project-lite"},
-    {name = "quick-error"},
-    {name = "rustc_version"},
-    {name = "rand"},
-    {name = "rand_chacha"},
-    {name = "rand_core"},
-    {name = "rand_hc"},
-    {name = "semver"},
-    {name = "sha-1"},
-    {name = "sha2"},
-    {name = "sha3"},
-    {name = "socket2"},
-    {name = "strsim"},
-    {name = "textwrap"},
-    {name = "wasi"},
-    {name = "winapi"},
-    {name = "windows"},
-    {name = "windows_aarch64_msvc" },
-    {name = "windows_i686_gnu" },
-    {name = "windows_i686_msvc" },
-    {name = "windows_x86_64_gnu" },
-    {name = "windows_x86_64_msvc" },
-    {name = "curve25519-dalek"},
-    {name = "hashbrown"},
-    {name = "universal-hash"},
-    {name = "poly1305"},
-    {name = "cipher"},
-    {name = "chacha20poly1305"},
-    {name = "chacha20"},
-    {name = "aead"},
+    {name = "base64", version = "=0.10.1"},
+    {name = "bech32", version = "=0.8.1"},
+    {name = "blake2", version = "=0.9.2"},
+    {name = "block-buffer", version = "=0.9.0"},
+    {name = "byteorder", version = "=0.5.3"},
+    {name = "digest", version = "=0.9.0"},
+    {name = "getrandom", version = "0.1.16"},
+    {name = "heck", version = "=0.3.3"},
+    {name = "parking_lot", version = "=0.11.2"},
+    {name = "parking_lot_core", version = "=0.8.5"},
+    {name = "quick-error", version = "=1.2.3"},
+    {name = "rustc_version", version = "=0.3.3"},
+    {name = "rand", version = "=0.4.6"},
+    {name = "rand", version = "=0.7.3"},
+    {name = "rand_core", version = "=0.3.1"},
+    {name = "rand_core", version = "=0.4.2"},
+    {name = "rand_core", version = "=0.5.1"},
+    {name = "rand_chacha", version = "=0.2.2"},
+    {name = "semver", version = "=0.11.0"},
+    {name = "sha-1", version = "=0.9.8"},
+    {name = "sha2", version = "=0.9.9"},
+    {name = "sha3", version = "=0.9.1"},
+    {name = "wasi", version = "=0.9.0"},
+    {name = "wasi", version = "=0.10.0"},
+    {name = "winapi", version = "=0.2.8"},
+    {name = "windows", version = "=0.32.0"},
+    {name = "windows_aarch64_msvc", version = "=0.32.0"},
+    {name = "windows_aarch64_msvc", version = "=0.34.0"},
+    {name = "windows_i686_gnu", version = "=0.32.0"},
+    {name = "windows_i686_gnu", version = "=0.34.0"},
+    {name = "windows_i686_msvc", version = "=0.32.0"},
+    {name = "windows_i686_msvc", version = "=0.34.0"},
+    {name = "windows_x86_64_gnu", version = "=0.32.0"},
+    {name = "windows_x86_64_gnu", version = "=0.34.0"},
+    {name = "windows_x86_64_msvc", version = "=0.32.0"},
+    {name = "windows_x86_64_msvc", version = "=0.34.0"},
+    {name = "curve25519-dalek", version = "=3.2.0"},
+    {name = "universal-hash", version = "=0.4.1"},
+    {name = "poly1305", version = "0.7.2"},
```

Comment:
Why doesn't this have the "=" inside the quotation marks too? Am I missing something obvious?

Reply:
By default the Cargo (and it seems that cargo deny uses the same behavior) version requirements are relaxed, for example:
```
1.2.3  :=  >=1.2.3, <2.0.0
1.2    :=  >=1.2.0, <2.0.0
1      :=  >=1.0.0, <2.0.0
```
So "=" means "use exactly this version".

Reply:
Yeah I get the idea. My question here perhaps a little more general, as to why we care for some of these and not others

Reply:
Is there a reason we treat the symmetric crypto stuff differently here?

Reply:
Not really, I have just forgot to put "=" to some crates. 🙃  Fixed.